README.md

```diff
@@ -13,3 +15,18 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+Answers:
+
+Misleading Analysis
+- Asymptotic analysis may be misleading in actual performance due to hardware limitations, such as the amount of threads that could be created on the machine might differ from what the algorithm requires.
+- Constant factors ignored in asymptotic analysis may be a significant enough factor to change performance significantly.
+- You might just be wrong about the found asymptotic analysis due to other factors, such as recursion being confusing and runtime analysis being generally difficult to grasp
+
+Input Size Asymptotic Complexity
+- Finding an element in a binary search tree of 1,000 elements could take 5 seconds. The same element in a tree of 10,000 elements will obviously take longer, but not by much. Search in an binary tree has the complexity of O(logn), where n is the number of nodes. O(log(1000)) = 3 where O(log(10,000)) = 4 and O(log(100,000)) = 5. Judging by how each solution changes by +1, and 1000 elements takes 5 seconds (3 * 2)-1 = 5, I would assume 10,000 elements takes (4 * 2)-1 = 7 seconds roughly. 
+
+Increased Asymptotic Complexity
+- The time could be quite longer than expected due to hardware errors such as allocating memory for the cache incorrectly or other background processes affecting cpu usage.
```

Comment:
Hardware errors and background processes would affect the smaller input size as well.